reporter-web-app/src/components/TableView.js

```diff
@@ -139,18 +139,7 @@ class TableView extends React.Component {
             filters: tableProjectFilterSelections,
             filteredValue: filteredInfo.id || null,
             onFilter: (value, webAppPackage) => webAppPackage.projectIndexes.has(value),
-            sorter: (a, b) => {
-                const idA = a.id.toUpperCase();
-                const idB = b.id.toUpperCase();
-                if (idA < idB) {
-                    return -1;
-                }
-                if (idA > idB) {
-                    return 1;
-                }
-
-                return 0;
-            },
+            sorter: (a, b) => a.id.localeCompare(b.id),
```

Comment:
If you want to maintain case-insensitive sorting here, you need to keep the `toUpperCase()` calls.

Reply:
I used the `toUpperCase()` as a hack for better performance but I don't need that anymore for `localCompare()` as is fast enough on modern browsers.